templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block title %}<title>Logged out</title>{% endblock %}
+{% block content %}
+  <p>You have been logged out.</p>
+  <a href="{% url 'login' %}">You can log in again</a>
```

Comment:
The URL tag should use the correct namespace for the login view. Update it to `{% url 'blog:login' %}` to ensure it resolves correctly within the blog app.